src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts

```diff
@@ -382,6 +382,11 @@ function showToolboxChanged(showing: boolean): void {
     if (currentTool) {
         if (showing) currentTool.hideTool();
         else currentTool.showTool();
+    } else {
```

Comment:
OK. We will probably want to show the previously shown tool instead, but that doesn't need to be part of this.